TreesForTravel/Redux/actions.js

```diff
@@ -0,0 +1,31 @@
+export const GET_FLIGHTS = "GET_FLIGHTS";
+export const POST_FLIGHT = "POST_FLIGHT";
+export const DELETE_FLIGHT = "DELETE_FLIGHT";
```

Comment:
Is this really necessary? Seems a little overkill just to save on quotation marks? (maybe I don't see the reason for this)